infra/src/main/java/com/redhat/service/bridge/infra/k8s/KubernetesClient.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.service.bridge.infra.k8s;
+
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.fabric8.kubernetes.api.model.networking.v1.Ingress;
+
+public interface KubernetesClient {
```

Comment:
I'm not sure about calling this `KubernetesClient`. I can just see us ending up in conversations where we say things like "our KubernetesClient, not the Fabric8 KubernetesClient" or "this is our KubernetesClient which wraps another KubernetesClient" 

I understand the intent, but I wonder if there is a better name for this class that stops us having slightly awkward conversations in the future?